src/ow1/role_limit.opy

```diff
@@ -0,0 +1,69 @@
+#!mainFile "../main.opy"
+
+#Global variables
+
+globalvar HeroListDamage 0
+globalvar HeroListTank 1
+globalvar HeroListSupport 2
+globalvar RoleLimitDamage 3
+globalvar RoleLimitTank 4
+globalvar RoleLimitSupport 5
+
+rule "[role_limit.opy]: Remove OW2 heroes and change Doomfist to DPS":
+    HeroListDamage = getDamageHeroes()
+    HeroListDamage.append([Hero.DOOMFIST])
+    HeroListDamage.remove([Hero.SOJOURN])
+    HeroListTank = getTankHeroes()
+    HeroListTank.remove([Hero.DOOMFIST,Hero.JUNKER_QUEEN,Hero.RAMATTRA])
+    HeroListSupport = getSupportHeroes()
+    HeroListSupport.remove([Hero.KIRIKO,Hero.LIFEWEAVER])
+    RoleLimitDamage = 2
+    RoleLimitTank = 2
+    RoleLimitSupport = 2
+
+
+rule "[role_limit.opy]: Role limit lockout damage":
+    @Event eachPlayer
+    @Condition len([player for player in getPlayers(eventPlayer.getTeam()) if player.getCurrentHero() in HeroListDamage]) >= RoleLimitDamage
+    @Condition eventPlayer.getCurrentHero() in HeroListDamage == false
```

Comment:
```suggestion
    @Condition (eventPlayer.getCurrentHero() in HeroListDamage) == false
```

Add parenthesis for safety and clarify; I forgot parenthesis in some condition in another code that took hours to debug.

Reply:
Done